<launch>
  
  <arg name="video_device" default="/dev/video2" />
  <arg name="camera_info_url" value="file://$(find perception_pkg)/camera_calibration/c505e.yaml" />

  <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device" value="$(arg video_device)"/>
    <param name="io_method" value="mmap"/>
    <param name="pixel_format" value="yuyv" />
    <param name="color_format" value="yuv422p" />
    <param name="camera_frame_id" value="usb_cam" />
    <param name="camera_name" value="c505e" />
    <param name="camera_info_url" value="$(arg camera_info_url)" />
    <param name="image_width" value="1280" />
    <param name="image_height" value="720" />
    <param name="framerate" value="30" />    
  </node>
</launch>
